### PR TITLE
Refactor/check 437 items should belong to one list

### PR DESCRIPTION
**Pull request by melsawy**

### CODE REVIEW

**Review by caiosba — APPROVED** [review]

File: src/app/components/media/MediaActionsBar.js [hunk 1]
```diff
@@ -250,22 +249,12 @@ class MediaActionsBarComponent extends Component {
       <div className={classes.root}>
         { media.archived === CheckArchivedFlags.NONE ?
           <div>
-            <AddProjectMediaToProjectAction
+            <MoveProjectMediaToProjectAction
```
Comment (amoedoamorim) [inline src/app/components/media/MediaActionsBar.js:252]: Perhaps we can now rename this to simply `MoveProjectMediaAction`

File: src/app/components/tag/TagMenu.js [hunk 2]
```diff
@@ -249,7 +249,7 @@ const TagMenuContainer = Relay.createContainer(withSetFlashMessage(TagMenuCompon
 // eslint-disable-next-line react/no-multi-comp
 class TagMenu extends React.PureComponent {
   render() {
-    const projectId = getCurrentProjectId(this.props.media.project_ids);
+    const projectId = getCurrentProjectId(this.props.media.project_id);
```
Comment (amoedoamorim) [inline src/app/components/tag/TagMenu.js:252]: Looks like the `getCurrentProjectId` and `getCurrentProject` helpers don't make sense anymore and can be removed.

File: src/app/helpers.js [hunk 3]
```diff
@@ -300,7 +300,7 @@ function getCurrentProjectId(projectIds) {
   const currentProjectId = window.location.pathname.match(/project\/([0-9]+)/);
   let projectId = currentProjectId ? parseInt(currentProjectId[1], 10) : null;
   if (projectId && projectIds) {
-    if (projectIds.indexOf(projectId) === -1) {
+    if (projectIds !== projectId) {
```
Comment (amoedoamorim) [inline src/app/helpers.js:303]: Looks like the getCurrentProjectId and getCurrentProject helpers don't make sense anymore and can be removed.

**Review by amoedoamorim — CHANGES_REQUESTED** [review]
Looks great! I just think we don't need those helpers anymore

File: src/app/components/media/MediaActionsBar.js (hunk 1 above)
Comment (melsawy) [reply src/app/components/media/MediaActionsBar.js]: Done and renamed to `MoveProjectMediaAction`

File: src/app/components/tag/TagMenu.js (hunk 2 above)
Comment (melsawy) [reply src/app/components/tag/TagMenu.js]: Removed both helpers

File: src/app/helpers.js (hunk 3 above)
Comment (melsawy) [reply src/app/helpers.js]: Removed both helpers

**Pull request merged**